LocalSettings.php

```diff
@@ -954,9 +954,6 @@
 	'wmgUseMediaWikiChat' => [
 		'default' => false,
 	],
-	'wmgUseMedik' => [
```

Comment:
it seems that you edited the previous version before I merged, so please add these lines back :)